src/firefly/js/api/ApiBuild.js

```diff
@@ -49,16 +49,28 @@ import {buildViewerApi} from './ApiViewer.js';
 import './ApiStyle.css';
 
 
-/**
+/*
```

Comment:
jsdoc has a command line option (or can be added in the configuration file) to specify the levels access of the documents.  In the source code, we need to use @access tag to specify the levels.   After this ticket is merged, I will send out the list of frequent used tags so each of us can edit the source codes to put proper tags.